### PR TITLE
Actions: switch from app to action for Stale issue/PR labeling

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

Instead of relying on an app, let's use actions to mark issues and labels as stale.
I've kept our stale options as close to the original as possible. The only option we're losing (because it is not available in the action) is the exemption for issues and PRs that are part of a project. I think it's an acceptable change.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* I'm not quite sure how to test this. Merge it and watch the action at work? :)

#### Proposed changelog entry for your changes:

* None

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          # Label to use when marking an issue / PR as stale
+          stale-pr-label: '[Status] Stale'
+          stale-issue-label: '[Status] Stale'
+          # Messages to display.
+          stale-issue-message: |
+            <p>This issue has been marked as stale. This happened because:</p>
+
+            <ul>
+              <li>It has been inactive in the past 6 months.</li>
+              <li>It hasn’t been labeled `[Pri] Blocker`, `[Pri] High`.</li>
```
Comment (anomiex) [inline .github/workflows/stale.yml:35]:
```suggestion
              <li>It hasn’t been labeled `[Pri] Blocker`, `[Pri] High`, etc.</li>
```

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            <p>No further action is needed. But it's worth checking if this ticket has clear
+            reproduction steps and it is still reproducible. Feel free to close this issue
+            if you think it's not valid anymore — if you do, please add a brief
+            explanation.</p>
+          stale-pr-message: |
+            <p>This PR has been marked as stale. This happened because:</p>
+
+            <ul>
+              <li>It has been inactive in the past 3 months.</li>
+              <li>It hasn’t been labeled `[Pri] Blocker`, `[Pri] High`.</li>
```
Comment (anomiex) [inline .github/workflows/stale.yml:47]:
```suggestion
              <li>It hasn’t been labeled `[Pri] Blocker`, `[Pri] High`, etc.</li>
```

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            </ul>
+
+            <p>No further action is needed. But it's worth checking if this PR has clear
+            testing instructions, is it up to date with master, and it is still valid.
+            Feel free to close this issue if you think it's not valid anymore — if you
```
Comment (anomiex) [inline .github/workflows/stale.yml:52]:
```suggestion
            Feel free to close this PR if you think it's not valid anymore — if you
```

File: .github/stale.yml [hunk 4]
```diff
@@ -1,63 +0,0 @@
...
-exemptMilestones: false
-
-# Label to use when marking an issue as stale
-staleLabel: "[Status] Stale"
-
-# Comment to post when closing a stale issue. Set to `false` to disable
-closeComment: false
-
-# Limit the number of actions per hour, from 1-30. Default is 30
-limitPerRun: 2
```
Comment (anomiex) [inline .github/stale.yml:31]: This got dropped. If you want to add it back, looks like the corresponding input is `operations-per-run`.

**Review by anomiex — APPROVED** [review]
Seems ok, some minor suggestions inline.

File: .github/stale.yml (hunk 4 above)
Comment (jeherve) [reply .github/stale.yml:31]: Ah yes, my bad. cf73d433761728929fe18497f39fae555b81dcf9

**Review by anomiex — APPROVED** [review]

**Pull request merged**